src/Components/PowerBIWidget/Internal/PowerBIWidgetBuilder/PowerBIWidgetBuilder.types.ts

```diff
@@ -0,0 +1,40 @@
+import { IStyle, IStyleFunctionOrObject, ITheme } from '@fluentui/react';
+import { IPage, IVisual } from 'powerbi-models';
+import { IPowerBIWidget } from '../../../../Models/Types/Generated/3DScenesConfiguration-v1.0.0';
+import { IWidgetBuilderFormDataProps } from '../../../ADT3DSceneBuilder/ADT3DSceneBuilder.types';
+
+export interface IPowerBIWidgetBuilderProps
+    extends IWidgetBuilderFormDataProps {
+    formData: IPowerBIWidget;
+    updateWidgetData: (widgetData: IPowerBIWidget) => void;
+    adapter?: IPowerBIWidgetBuilderAdapter;
+    /**
+     * Call to provide customized styling that will layer on top of the variant rules.
+     */
+    styles?: IStyleFunctionOrObject<
+        IPowerBIWidgetBuilderStyleProps,
+        IPowerBIWidgetBuilderStyles
+    >;
+}
+
+export interface IPowerBIWidgetBuilderStyleProps {
+    theme: ITheme;
+}
+export interface IPowerBIWidgetBuilderStyles {
+    root: IStyle;
+    header?: IStyle;
```

Comment:
are these styles really optional? Usually the styles we define for components are required so I'm curious if this is on purpose?